### PR TITLE
Ny workspace

### PR DESCRIPTION
**Pull request by mertcihankurel**
### New York Times Article page

**İn this project:**
1-) Used two fixed navbar for top and bottom.
2-) İcons replaced with relative positioning.
3-) Added borders around of categories divisions.
4-) Added negative margin for figure descriptions.
5-)Added embedded video and images.
6-)Navbar z-index fixed
7-)Bottom spaces deleted
8-) Given equal padding button elements
9-) Linters errors fixed

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job @mcihadkurel your project looks good :clap: :tophat: , but your repository is terrible you need to make it look professional before you request any code review please do that and then request another code review. :hammer: :hammer_and_pick: :wrench: :toolbox: 

Friendly reminder: You should make sure that your repo looks professional before review. Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

## Happy Coding :smiley: 
_If you have any question or you felt stuck at any point feel free to contact me on slack_

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+
```
Comment (elmejdki) [inline README.md]: please remove these extras links

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job so far @mcihadkurel :clap: :clap: , your project looks great near to the original page but you need to make some micro :microscope:  changes :hammer: :toolbox:, in order to get it approved. :heavy_check_mark: 

## status: require changes. :recycle: 

- [ ] please fix the header:
### clone:
image

### It should look like this:
image

- [ ] Wrap the new york times logo inside a link tag.
- [ ] the social media links on the bottom overlap over the top menu please use a bigger z-index for the top menu to fix it:
image

- [ ] **[OPTIONAL]** add some hover styling on the links, for example, make the links change colors when someone hovers over them:
image

- [ ] you have a huge space on the button please remove it:
image

- [ ] this time I will not judge you about it, but next time please write good commit messages:
image
here is a really good [article](https://chris.beams.io/posts/git-commit/) that explains how to write good commit messages, please give it a try.

### Please make all the required changes and then request another code review.
## Happy Coding :tophat: 
_If you have any questions don't hesitate to contact me on slack._

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @mcihadkurel :wave:
-
You have done a splendid job! :tada: Keep it up.
We, however, do have some few changes remaining to carry out before we proceed (see review comment).
##
Lests fix the following minor issues highlighted below 👇 
|[Editor's Pick](https://raw.githack.com/mcihadkurel/nyTimes-article/ny-workspace/index.html) section|[most popular](https://raw.githack.com/mcihadkurel/nyTimes-article/ny-workspace/index.html) section|
|:---:|:---:|
|clone|clone-2|

- [ ] the editor's pick section is not well paced. Kindly reposition it towards the middle of the page. 
- [ ] Work on the most popular section.
##
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by mertcihankurel** [comment]
Hi. Thank you for your reply. This project has done for laptop screen sizes, and as I now; it's not required to fix this project for all kinds of screen sizes. Please check this screenshots below. 
nyss1
nyss2

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
### HELLO @mcihadkurel :wave:  AM PETER A TSE AND I WILL BE REVIEWING YOUR PROJECT LETS TAKE A LOOK SHALL WE :eye: 

Great work and you have done everything requested by the previous TSE. One thing prevented me from approving this project kindly fix :arrow_down: 

- [ ] I can see you have the files for linting. However, it seems you forgot to enable it in your repo kindly do that.

**IF YOU HAVE ANY QUESTION FEEL FREE TO REACH OUT TO ME VIA SLACK** :smile: 

### HAPPY CODING :computer:

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved :medal_sports: 

### Hi @mcihadkurel :hand: 

### You did great with the project :+1:  You can merge this branch into master

**Pull request merged**